ironfish-cli/src/utils/tar.ts

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { spawn } from 'child_process'
+import path from 'path'
+import tar from 'tar'
+
+async function zipDir(source: string, dest: string, excludes: string[] = []): Promise<void> {
+  const sourceDir = path.dirname(source)
+  const sourceFile = path.basename(source)
+  const excludeSet = new Set(excludes)
+
+  await tar.create(
+    {
+      gzip: true,
+      file: dest,
+      C: sourceDir,
+      filter: function (path) {
+        if (excludeSet.has(path)) {
+          return false
+        } else {
+          return true
+        }
+      },
+    },
+    [sourceFile],
+  )
+}
+
+function unzipTar(source: string, dest: string): Promise<number | null> {
```

Comment:
we could also implement this using the `'tar'` module --`importSnapshot` implements unzip that way

Reply:
Yeah that would require code change, I was just moving code around to re-use it. I hope someone else will do that.